src/types.ts

```diff
@@ -1,12 +1,84 @@
+export interface Record<T> {
```

Comment:
Ohhhhh! Let's not do this. `Record` is already a TypeScript type. That's why I said the comment above. Maybe call it `AirTableRecord<T>`?